### PR TITLE
Message for Invalid forestType 50a 

### PR DESCRIPTION
**Pull request by angsherpa232**

### CODE REVIEW

File: src/components/ProjectionError.js [hunk 1]
```diff
@@ -4,32 +4,22 @@ import { useTranslation } from 'react-i18next';
 import { Message } from 'semantic-ui-react';
 import styles from './projectionError.module.css';
 
-const requiredFieldsForProjection = [
```
Comment (friedjoff) [inline src/components/ProjectionError.js]: Why did you remove the required field validation?

File: src/components/ProjectionError.js [hunk 2]
```diff
@@ -4,32 +4,22 @@ import { useTranslation } from 'react-i18next';
...
 
   const { t } = useTranslation();
-  const missingFields = requiredFieldsForProjection
-    .filter(field => mapLocation[field] === undefined)
-    .map(field => t(`${field}.label`));
 
-  return projectionMode === 'm' &&
-    mapLocation.coordinate &&
-    missingFields.length >= 1 ? (
+  return projectionMode === 'm' && mapLocation.forestType === '50a' ? (
```
Comment (friedjoff) [inline src/components/ProjectionError.js]: This should be handled in a more general way. Invalid forest types should not be hardcoded. We need to catch the error from the project function and handle it there.

File: src/components/ProjectionError.js (hunk 1 above)
Comment (angsherpa232) [reply src/components/ProjectionError.js]: Because last time it was discussed and Uli suggested so in the ticket.

**Comment by friedjoff** [comment]
Will be easier to start a new implementation based on master which will catch errors from the project function.
